tasks/main.yml

```diff
@@ -1,43 +1,49 @@
 ---
+- name: Register php version
+  shell: command -v php && php --version
+  register: php_version
+  changed_when: false
+  ignore_errors: true
```

Comment:
maybe it should fail if php is not installed?

Reply:
No, `fail` not working.

Reply:
You mean it fails even if PHP is installed?

Reply:
![image](https://user-images.githubusercontent.com/26375175/34434760-51d8d2aa-ecc3-11e7-98a7-49f7c9573d2b.png)
no matter php installed or not.